Makefile

```
@@ -0,0 +1,191 @@
+		fish_indent --check "$$1" || { \
+			echo "Formatting issues found in $$1"; exit 1; }' \
+		sh {} \;
+	@echo "Validating Fish syntax..."
+	@fish -n functions/*.fish completions/*.fish 2>/dev/null || { \
+		echo "Syntax errors found in Fish files"; \
+		exit 1; \
+	}
+	@echo "Fish files passed linting!"
+
+# Lint Markdown files
+lint-markdown:
```

checkmake (0.3.2)

[warning] Target body for "lint-markdown" exceeds allowed length of 5 lines (10).

(maxbodylength)